lib/fuse_versionscript

```diff
@@ -166,6 +166,8 @@ FUSE_3.7 {
 	global:
 		fuse_set_log_func;
 		fuse_log;
+		fuse_passthrough_open;
+		fuse_passthrough_close;
 } FUSE_3.4;
```

Comment:
That is not right here and will cause issues for distributions. Could you please put it under FUSE_3.17? We might need to change it again to 3.18 if 3.17 gets released before (it probably will because of the ABI issue) - hmm, maybe wait a few days, I hope I get a bit time tomorrow morning to look into that.

Reply:
Should I use FUSE_3.17 { } FUSE_3.4;?
What does FUSE_3.4 mean in this context?

Reply:
https://ftp.gnu.org/old-gnu/Manuals/ld-2.9.1/html_mono/ld.html#SEC25

To my understanding "FUSE_3.4" is what the new version depends on. Given that we are at ABI version 3.12 it should be `FUSE_3.17 { } FUSE_3.12;`, afaik.

With the possibility that 3.17 needs to be replaced by 3.18 later this week.

Reply:
ok, it is a mistake that the last section is FUSE_3.12 { } FUSE_3.4 ? 
should it not be FUSE_3.12 {} FUSE_3.7 ?

Reply:
Ouch, I didn't see that, yeah, I had used the wrong dependency value. I'm not sure if I should change it now. Maybe all is fine, as the fuse_session_loop dependency symbols are defined in 3.2.

Reply:
@bsbernd I see the libfuse version was increased to 3.17.0.
Did you mean that the passthrough feature should increment the libfuse version to 3.18.0?

Reply:
It is not finished yet, the version is was just increased to prepare the branch already.
I have created a separate patch/PR to add in the libfuse version an application was compiled against (https://github.com/libfuse/libfuse/pull/942), should be complete now, but I didn't have time for manual testing yet. I hope this weekend.  After that will add in auto-correct of the ABI issues for < 3.17.0 and then we should be good to release.